@here/harp-examples/src/performance_benchmark.ts

```diff
@@ -631,7 +648,7 @@ export namespace PerformanceBenchmark {
                 "6": 6,
                 "8": 8
             },
-            PhasedLoading: true,
+            PhasedLoading: false,
```

Comment:
This should have already been removed? Maybe make a ticket for this.

Reply:
true, will remove in a follow up commit